Dev Fund ZIP: Hybrid Deferred Dev Fund (#860)
zcash/zips: draft-zip-hybrid-deferred-devfund.md

# Hybrid Deferred Dev Fund ZIP
  

    ZIP: Unassigned {numbers are assigned by ZIP editors}
    Title: Hybrid Deferred Development Fund
    Owners: Jason McGee <aquietinvestor@gmail.com>
            Skylar Saveland <skylar@free2z.com>
    Credits: Daira Emma Hopwood, Peacemonger, Gguy
    Status: Draft
    Category: Consensus | Process

>     Created: 2014-06-25

Reviewer: 2024 (?)

* **Improving Funding Efficiency**: Allocating resources directly to projects rather than organizations reduces inefficiencies and better aligns with community priorities.

* **Limited Runway**: ZCG does not have the financial runway that ECC/BP and ZF have. As such, allocating ongoing funding to ZCG will help ensure the Zcash ecosystem has an active grants program. 

* **Promoting Decentralization**: Allocating a portion of the Dev Fund to Zcash Community Grants ensures small teams continue to receive funding to contribute to Zcash. Allowing the Dev Fund to expire, or putting 100% into a lockbox, would disproportionally impact grant recipients. This hybrid approach promotes decentralization and the growth of independent development teams. 

* **Moving Towards a Non-Direct Funding Model**: There is strong community support for a non-direct Dev Fund funding model.  Allocating funds to a Hybrid Deferred Dev Fund Lockbox gives the community time to research and implement such a model while continuing to fund grant recipients.  

## Requirements

> **In-Protocol Lockbox Creation**: The ZIP must define the creation of an in-protocol lockbox where 20% of the Zcash block reward will be allocated post-November 2024 halving. This lockbox will securely store funds until a disbursement mechanism is established in a future ZIP.

Reviewer: Is this correct(?) I had understood that from the 20% total, only 12% goes into the in-protocol lockbox, while the other 8% go to ZCG as either direct recipient/ or indirect through ZF or FPF as administrator.

Author: 8% is consistent with ZCG's current allocation of the block reward (i.e. 40% of the Dev Fund).

Reviewer: My question is about where the ZEC will flow (I understand that the flow is split into 2 receiving entities: ZCG/FPF and the Lockbox).  But yes, thanks for clarifying the intention behind the increased relative ZCG allocation.

Author: Re: The ZCG portion: The ZEC will flow to FPF, but will be restricted funding that can only be used for ZCG grants. FPF will charge an admin fee to ZCG to support the program.

# Hybrid Deferred Dev Fund ZIP
  

    ZIP: Unassigned {numbers are assigned by ZIP editors}
    Title: Hybrid Deferred Development Fund
    Owners: Jason McGee <aquietinvestor@gmail.com>
            Skylar Saveland <skylar@free2z.com>
    Credits: Daira Emma Hopwood, Peacemonger, Gguy
    Status: Draft
    Category: Consensus | Process
    Created: 2014-06-25
    License: MIT
    Pull-Request: <https://github.com/zcash/zips/pull/???>

## Terminology

The key words "MUST", "REQUIRED", "MUST NOT", "SHOULD", and "MAY" in this document are to be interpreted as described in BCP 14 [#BCP14]_ when, and only when, they appear in all capitals.

{Avoid duplicating definitions from other ZIPs. Instead use wording like this:}

The terms "Mainnet" and "Testnet" in this document are to be interpreted as defined in the Zcash protocol specification [#protocol-networks]_.

The term "full validator" in this document is to be interpreted as defined in the Zcash protocol specification [#protocol-blockchain]_.

> The terms below are to be interpreted as follows:
> 

Reviewer 2: ```suggestion
```

The term "network upgrade" in this document is to be interpreted as described in ZIP 200 6 and the Zcash Trademark Donation and License Agreement (4 or successor agreement).

The terms "block subsidy" and "halving" in this document are to be interpreted as described in sections 3.10 and 7.8 of the Zcash Protocol Specification. 2

"Zcash Foundation", also called "ZF", refers to the 501(c)(3) nonprofit corporation of that name.

"Section 501(c)(3)" refers to that section of the U.S. Internal Revenue Code (Title 26 of the U.S. Code).

"Community Advisory Panel" refers to the panel of community members assembled by the Zcash Foundation. 

The terms "Testnet" and "Mainnet" are to be interpreted as described in section 3.12 of the Zcash Protocol Specification

> Useful ZIPs to reference:
> * [ZIP 214](https://zips.z.cash/zip-0214) (funding streams for the current dev fund)
> * [ZIP 1014](https://zips.z.cash/zip-1014) (process issues for the current dev fund)
> * [ZIP 207](https://zips.z.cash/zip-0207) (how funding streams work)

Reviewer 2: ```suggestion
```

# Hybrid Deferred Dev Fund ZIP

    ZIP: Unassigned {numbers are assigned by ZIP editors}
    Title: Hybrid Deferred Development Fund
    Owners: Jason McGee <aquietinvestor@gmail.com>
            Skylar Saveland <skylar@free2z.com>
            Peacemonger < >

>             Gguy <babygguy@gmail.com

Reviewer 3: ```suggestion
            Gguy <babygguy@gmail.com>
```

Author: @NerdGGuy - You are now listed as an owner.

    Credits: Daira Emma Hopwood, Str4d, Kris Nuttycomb
    Status: Draft
    Category: Consensus | Process
    Created: 2024-06-25
    License: MIT
    Pull-Request: <https://github.com/zcash/zips/pull/???>

## Terminology

The key words "MUST", "REQUIRED", "MUST NOT", "SHOULD", and "MAY" in this
document are to be interpreted as described in BCP 14 [#BCP14]_ when, and only
when, they appear in all capitals.

The terms "Mainnet" and "Testnet" in this document are to be interpreted as
defined in the Zcash protocol specification [#protocol-networks]_.

The term "full validator" in this document is to be interpreted as defined in
the Zcash protocol specification [#protocol-blockchain]_.

The term "network upgrade" in this document is to be interpreted as described
in ZIP 200 6 and the Zcash Trademark Donation and License Agreement (4 or
successor agreement).

The terms "block subsidy" and "halving" in this document are to be interpreted
as described in sections 3.10 and 7.8 of the Zcash Protocol Specification. 2

"Zcash Foundation", also called "ZF", refers to the 501(c)(3) nonprofit
corporation of that name.

> 

Reviewer 3: ```suggestion

"Financial Privacy Foundation", also called "FPF", refers to the independent Cayman Islands-Incorporated non-profit organization of that name.

```

   which may be disbursed for expenses reasonably related to the administration
   of the ZCG program. The amount of funds allocated to the Discretionary
   Budget SHALL be decided by the ZF’s Community Advisory Panel or successor
   process. Any disbursement of funds from the Discretionary Budget MUST be
   approved by the ZCG Committee. Expenses related to the administration of the
   ZCG program include, without limitation the following:
    * Paying third party vendors for services related to domain name
      registration, or the design, website hosting and administration of
      websites for the ZCG Committee.
    * Paying independent consultants to develop requests for proposals that
      align with the ZCG program.
    * Paying independent consultants for expert review of grant applications.
    * Paying for sales and marketing services to promote the ZCG program.
    * Paying third party consultants to undertake activities that support the
      purpose of the ZCG program.
    * Reimbursement to members of the ZCG Committee for reasonable travel
      expenses, including transportation, hotel and meals allowance.

> 
> 9. A portion of the Discretionary Budget MAY be allocated to provide reasonable
>    compensation to members of the ZCG Committee. Committee member compensation
>    SHALL be limited to the hours needed to successfully perform their positions
>    and MUST align with the scope and responsibilities of their roles. The
>    allocation and distribution of compensation to committee members SHALL be
>    administered by the ZF or FPF. The compensation rate and hours for committee
>    members SHALL be determined by the ZF’s Community Advisory Panel or
>    successor process.
> 
> 10. The ZCG Committee’s decisions relating to the allocation and disbursement
>     of funds from the Discretionary Budget will be final, requiring no approval
>     from the ZF or FPF Board, but are subject to veto if the Foundation judges
>     them to violate U.S. law or the ZF’s reporting requirements and other
>     (current or future) obligations under U.S. IRS 501(c)(3), or, in the case
>     of FPF, other similar requirements of the relevant jurisdiction.
> 
> ZF or FPF SHALL recognize the ZCG slice of the Dev Fund as a Restricted Fund
> donation under the above constraints (suitably formalized), and keep separate
> accounting of its balance and usage under its Transparency and Accountability
> obligations defined below.
> 
> ZF or FPF SHALL strive to define target metrics and key performance indicators,
> and the ZCG Committee SHOULD utilize these in its funding decisions.
> 
> **Direct Grant Option**
> 
> It may be deemed better, operationally or legally, if ZCG funds are not
> accepted and disbursed by ZF or FPF, but rather directly assigned to the
> grantees. Thus, the following mechanism MAY be used in perpetuity for some or
> all grantees:
> 
> Prior to each network upgrade, based on the ZCG Committee’s recommendation, the
> Foundation SHALL publish a list of grantees’ addresses and the total number of
> Dev Fund ZEC per block they should receive. ECC and ZF SHALL implement this
> list in any implementations of the Zcash consensus rules they maintain. This
> decision will then be, effectively, ratified by the miners as the network
> upgrade activates.
> 
> **Furthering Decentralization**
> 
> ZF or FPF SHALL conduct periodic reviews of the
> organizational structure, performance, and effectiveness of the ZCG program and
> committee, taking into consideration the input and recommendations of the ZCG
> Committee. As part of these periodic reviews, ZF or FPF MUST commit to
> exploring the possibility of transitioning ZCG into an independent organization
> if it is economically viable and it aligns with the interests of the Zcash
> ecosystem and prevailing community sentiment.
> 
> In any transition toward independence, priority SHALL be given to maintaining
> or enhancing the decentralization of the Zcash ecosystem. The newly formed
> independent organization MUST ensure that decision-making processes remain
> community-driven, transparent, and responsive to the evolving needs of the
> Zcash community and ecosystem. In order to promote geographic decentralization,
> the new organization SHOULD establish its domicile outside of the United
> States.
> 
> **Transparency and Accountability**
> 
> ZF or FPF accept the following obligations in this section on behalf of ZCG:

Reviewer 3: Here are the changes I believe would be required if this proposal commits to FPF administration.

```suggestion
The funds SHALL be received and administered by the
Financial Privacy Foundation (FPF). FPF MUST disburse them for grants and
expenses reasonably related to the administration of the ZCG program, but
subject to the following additional constraints:

1. These funds MUST only be used to issue grants to external parties that are
   independent of FPF, and to pay for expenses reasonably related to the
   administration of the ZCG program. They MUST NOT be used by FPF for
   its internal operations and direct expenses not related to the
   administration of grants or the grants program.

2. ZCG SHOULD support well-specified work proposed by the grantee, at
   reasonable market-rate costs. They can be of any duration or ongoing without
   a duration limit. Grants of indefinite duration SHOULD have semiannual
   review points for continuation of funding.

3. Priority SHOULD be given to major grants that bolster teams with substantial
   (current or prospective) continual existence, and set them up for long-term
   success, subject to the usual grant award considerations (impact, ability,
   risks, team, cost-effectiveness, etc.). Priority SHOULD be given to major
   grants that support ecosystem growth, for example through mentorship,
   coaching, technical resources, creating entrepreneurial opportunities, etc.
   If one proposal substantially duplicates another’s plans, priority SHOULD be
   given to the originator of the plans.

4. The ZCG committee SHOULD be restricted to funding projects that further the
   Zcash cryptocurrency and its ecosystem (which is more specific than
   furthering financial privacy in general) as permitted by FPF
   and any relevant jurisdictional requirements.

5. ZCG awards are subject to approval by a five-seat ZCG Committee. The ZCG
   Committee SHALL be selected by the ZF’s Community Advisory Panel or a
   successor process (e.g. as established by FPF). Elections SHALL be staggered
   to ensure continuity within the Committee.

6. The ZCG Committee’s funding decisions will be final, requiring no approval
   from the FPF Board, but are subject to veto if the FPF judges them to
   violate any relevant laws or other (current or future) obligations.

7. ZCG Committee members SHALL have a one-year term and MAY sit for reelection.
   The ZCG Committee is subject to the same conflict of interest policy that
   governs the ZF Board of Directors (i.e. they MUST recuse themselves when
   voting on proposals where they have a financial interest). At most one
   person with association with the BP/ECC, at most one person with
   association with the ZF, and at most one person with association with FPF
   are allowed to sit on the ZCG Committee.
   “Association” here means: having a financial interest, full-time employment,
   being an officer, being a director, or having an immediate family
   relationship with any of the above. The ZF SHALL continue to operate the
   Community Advisory Panel and SHOULD work toward making it more
   representative and independent (more on that below). Similarly, FPF should
   also endeavor to establish its own means of collecting community sentiment
   for the purpose of administering ZCG elections.

8. A portion of the ZCG Slice shall be allocated to a Discretionary Budget,
   which may be disbursed for expenses reasonably related to the administration
   of the ZCG program. The amount of funds allocated to the Discretionary
   Budget SHALL be decided by the ZF’s Community Advisory Panel or successor
   process. Any disbursement of funds from the Discretionary Budget MUST be
   approved by the ZCG Committee. Expenses related to the administration of the
   ZCG program include, without limitation the following:
    * Paying third party vendors for services related to domain name
      registration, or the design, website hosting and administration of
      websites for the ZCG Committee.
    * Paying independent consultants to develop requests for proposals that
      align with the ZCG program.
    * Paying independent consultants for expert review of grant applications.
    * Paying for sales and marketing services to promote the ZCG program.
    * Paying third party consultants to undertake activities that support the
      purpose of the ZCG program.
    * Reimbursement to members of the ZCG Committee for reasonable travel
      expenses, including transportation, hotel and meals allowance.

9. A portion of the Discretionary Budget MAY be allocated to provide reasonable
   compensation to members of the ZCG Committee. Committee member compensation
   SHALL be limited to the hours needed to successfully perform their positions
   and MUST align with the scope and responsibilities of their roles. The
   allocation and distribution of compensation to committee members SHALL be
   administered by the FPF. The compensation rate and hours for committee
   members SHALL be determined by the ZF’s Community Advisory Panel or
   successor process.

10. The ZCG Committee’s decisions relating to the allocation and disbursement
    of funds from the Discretionary Budget will be final, requiring no approval
    from the FPF Board, but are subject to veto if the FPF judges
    them to violate laws or FPF reporting requirements and other
    (current or future) obligations.
    
FPF SHALL recognize the ZCG slice of the Dev Fund as a Restricted Fund
donation under the above constraints (suitably formalized), and keep separate
accounting of its balance and usage under its Transparency and Accountability
obligations defined below.

FPF SHALL strive to define target metrics and key performance indicators,
and the ZCG Committee SHOULD utilize these in its funding decisions.

**Direct Grant Option**

It may be deemed better, operationally or legally, if ZCG funds are not
accepted and disbursed by FPF, but rather directly assigned to the
grantees. Thus, the following mechanism MAY be used in perpetuity for some or
all grantees:

Prior to each network upgrade, based on the ZCG Committee’s recommendation, the
FPF SHALL publish a list of grantees’ addresses and the total number of
Dev Fund ZEC per block they should receive. ECC and ZF SHALL implement this
list in any implementations of the Zcash consensus rules they maintain. This
decision will then be, effectively, ratified by the miners as the network
upgrade activates.

**Furthering Decentralization**

FPF SHALL conduct periodic reviews of the
organizational structure, performance, and effectiveness of the ZCG program and
committee, taking into consideration the input and recommendations of the ZCG
Committee. As part of these periodic reviews, FPF MUST commit to
exploring the possibility of transitioning ZCG into an independent organization
if it is economically viable and it aligns with the interests of the Zcash
ecosystem and prevailing community sentiment.

In any transition toward independence, priority SHALL be given to maintaining
or enhancing the decentralization of the Zcash ecosystem. The newly formed
independent organization MUST ensure that decision-making processes remain
community-driven, transparent, and responsive to the evolving needs of the
Zcash community and ecosystem. In order to promote geographic decentralization,
the new organization SHOULD establish its domicile outside of the United
States.

**Transparency and Accountability**

FPF accept the following obligations in this section on behalf of ZCG:
```

* Publication of the ZCG Dashboard, providing a snapshot of ZCG’s current
  financials and any disbursements made to grantees.
* Bi-weekly meeting minutes documenting the decisions made by the ZCG committee
  on grants.
* Quarterly reports, detailing future plans, execution on previous plans, and
  finances (balances, and spending broken down by major categories).
* Annual detailed review of the organization performance and future plans.
* Annual financial report (IRS Form 990, or substantially similar information).

BP, ECC, ZF, FPF, ZCG and grant recipients MUST promptly disclose any security
or privacy risks that may affect users of Zcash (by responsible disclosure
under confidence to the pertinent developers, where applicable).

All substantial software whose development was funded by the Dev Fund SHOULD be
released under an Open Source license (as defined by the Open Source Initiative
5), preferably the MIT license.

**Enforcement**

> ZF or FPF MUST contractually commit to fulfill these obligations on behalf of

Reviewer 3: ```suggestion
FPF MUST contractually commit to fulfill these obligations on behalf of
```

# Hybrid Deferred Dev Fund ZIP

    ZIP: Unassigned {numbers are assigned by ZIP editors}
    Title: Hybrid Deferred Development Fund
    Owners: Jason McGee <aquietinvestor@gmail.com>
            Skylar Saveland <skylar@free2z.com>
    Credits: Daira Emma Hopwood, Peacemonger, Gguy
    Status: Draft
    Category: Consensus | Process

>     Created: 2014-06-25

Reviewer 3: ```suggestion
    Created: 2024-06-25
```

Author: I previously updated this one, so we're good here.

  legal risks.

* **Improving Funding Efficiency**: Allocating resources directly to projects
  rather than organizations reduces inefficiencies and better aligns with
  community priorities.

* **Limited Runway**: ZCG does not have the financial runway that ECC/BP and ZF
  have. As such, allocating ongoing funding to ZCG will help ensure the Zcash
  ecosystem has an active grants program.

* **Promoting Decentralization**: Allocating a portion of the Dev Fund to Zcash
  Community Grants ensures small teams continue to receive funding to
  contribute to Zcash. Allowing the Dev Fund to expire, or putting 100% into a
  lockbox, would disproportionally impact grant recipients. This hybrid
  approach promotes decentralization and the growth of independent development
  teams.

* **Moving Towards a Non-Direct Funding Model**: There is strong community
  support for a non-direct Dev Fund funding model.  Allocating funds to a
  Hybrid Deferred Dev Fund Lockbox gives the community time to research and
  implement such a model while continuing to fund grant recipients.

## Requirements

**In-Protocol Lockbox Creation**: The ZIP must define the creation of an

> in-protocol lockbox where 20% of the Zcash block reward will be allocated

Reviewer 4: This is incompatible with the abstract or the specification (which imply that 12% goes into the lockbox). For ease of reading, just remove the percentage from here:
```suggestion
in-protocol lockbox where a percentage of the Zcash block reward will be allocated
```

protect against unauthorized access or misuse of funds. This includes ensuring
the integrity of the future decentralized voting/signature mechanism.

**Future Adaptability**: The lockbox design should be flexible and adaptable to
accommodate the disbursement mechanism defined in a future ZIP. This ensures
that the system can evolve based on feedback and new developments in the Zcash
ecosystem.

**Non-Direct Funding Model**: This ZIP recognizes there is strong community
support for a non-direct funding model.  As such, ECC and ZF MUST collaborate
with the Zcash community to research and explore the establishment of a
non-direct funding model. The research should consider potential designs as
well as possible legal and regulatory risks.

**ZCG Slice**: The ZCG slice should encourage decentralization and support
independent teams dedicated to Zcash. Major funding decisions should be based,
to the extent feasible, on inputs from domain experts and pertinent
stakeholders.

**Value of ZEC**: In case the value of ZEC jumps, ZCG should not wastefully use
excessive amounts of funds. Conversely, given market volatility and eventual

> halvings, it is desirable to create rainy-day reserves.

Reviewer 4: These sections should be in a process subsection of the specification. Requirements constrain the ZIP itself, not anyone "implementing" the ZIP.

Author: @str4d Apologies, but I don't fully understand your comment here. Can you clarify or (even better) would you be able to propose a change that I can review and accept?

**Funding Stream**: The ZCG funding stream should conform to the standards of
ZIP 207 and ZIP 214.

## Specification

Consensus changes implied by this specification are applicable to the Zcash
Mainnet. Similar (but not necessarily identical) consensus changes SHOULD be
applied to the Zcash Testnet for testing purposes.

### Hybrid Deferred Dev Fund Allocation

The Zcash Development Fund SHALL be enacted in Network Upgrade 6 and last for 2
years. The Dev Fund would consist of 20% of the block subsidies, split into 2
slices:

> * 60% to a deferred Dev Fund “lockbox,” which will be allocated in the future
>   once consensus is reached on fund allocation or a new non-direct funding
>   model is established and implemented.

Reviewer 4: Process constraints and consensus constraints should be separate. This is a consensus constraint:
```suggestion
* 60% to a deferred Dev Fund “lockbox”.
```

The process constraint needs to be described in a separate subsection of the specification, in order for this ZIP to be semantically stable.

* 40% to Zcash Community Grants to fund independent teams in the Zcash ecosystem.

### Deferred Dev Fund Lockbox

<u>Deferred Development Fund Chain Value Pool Balance</u>

A new development fund value pool balance, denominated in zatoshis, will be
created. This balance will be initialized to zero at the NU6 activation block.

<u>Coinbase Transaction Adjustments</u>

> For coinbase transactions, a specified proportion of the block subsidy will be
> added to the deferred development fund chain value pool balance and subtracted
> from the transparent transaction value pool. This subtraction will occur after
> block reward value is logically added to the transparent value pool. The exact
> proportions (numerator/denominator) of the block subsidy allocated to the
> deferred development fund must be defined to ensure the correct allocation at
> every block.

Reviewer 4: This part of the spec is incomplete, and includes an incorrect line. For the purposes of today's diff, remove the incorrect line; the remainder of this section is sufficient for today's deadline.
```suggestion
For coinbase transactions, a specified proportion of the block subsidy will be
added to the deferred development fund chain value pool balance and subtracted
from the transparent transaction value pool. The exact
proportions (numerator/denominator) of the block subsidy allocated to the
deferred development fund must be defined to ensure the correct allocation at
every block.
```

<u>Proportion Specification</u>

The proportions of the block subsidy allocated to the deferred development fund
and any existing funding streams must add up to the intended total at every
block. Before NU6 activation, the actual percentage allocated to the deferred

> development fund will be zero. After NU6 activation, |percentage| of the block

Reviewer 4: The abstract, and the beginning of the specification, have defined percentages; here this appears to instead imply a parameter.

For the purposes of today's deadline:
- If this is intended to be a parameter, define the parameter at the start of the spec section (before its first use), and use it consistently everywhere.
- If it is not intended to be a parameter, replace this with 12% to be consistent with the rest of the ZIP draft.

Author: It should be 12%, consistent with the rest of the ZIP draft.

reward will be allocated to the deferred development fund.

> <u>Backwards Compatibility</u>
> 
> Full validators can implement the reserve consensus rules without NU gating.
> Prior to NU6 activation, the percentage allocated to the reserve pool can be
> set to zero or be defined as an empty set, similar to existing funding streams.
> Additionally, another implementation option would be to define a new type of
> funding stream that directs the value to the in-protocol development fund
> instead of constructing a UTXO. This approach allows for flexibility in
> handling the allocation and ensures compatibility with existing mechanisms.
> 

Reviewer 4: This should be in a rationale section, if included. For now, sufficient to delete from the specification section.
```suggestion
```

<u>Reserve Pool Mechanism</u>

The reserve pool will be a container for the issued supply, distinct from
existing pools. It will not be subdivided into individually spendable notes or
coins. The lockbox mechanism ensures that funds are accumulated and securely
stored until a future ZIP defines the disbursement mechanism.

> <u>Security and Integrity</u>
> 
> Robust security measures must be implemented to protect the deferred
> development fund from unauthorized access or misuse. The integrity of the
> lockbox and its interaction with the Zcash protocol must be ensured.
> 

Reviewer 4: This does not belong in the spec; it is a requirement on the spec (and is already present in the requirements section).
```suggestion
```

> ### Zcash Community Grants (ZCG)

Reviewer 4: @nuttycom to check how this section differs froim ZIP 1014, if at all.

This slice of the Dev Fund is intended to fund independent teams entering the
Zcash ecosystem, to perform major ongoing development (or other work) for the
public good of the Zcash ecosystem, to the extent that such teams are available
and effective. The Zcash Community Grants (ZCG) Committee is given the
discretion to allocate funds not only to major grants, but also to a diverse
range of projects that advance the usability, security, privacy, and adoption
of Zcash, including community programs, dedicated resources, and other projects
of varying sizes.

The funds SHALL be received and administered by the Zcash Foundation (ZF) or

> Financial Privacy Foundation (FPF). ZF or FPF MUST disburse them for grants and

Reviewer 4: The choice of ZF or FPF here would need to be considered a parameter for the purposes of today's deadline; it needs to be decided upon by the next deadline, as it cannot be either-or in the final ZIP.

Author: Okay, understood. I believe it will be FPF, similar to what's reflected in Dodger's proposal. I will confirm and follow up in the next day or two.

list in any implementations of the Zcash consensus rules they maintain. This
decision will then be, effectively, ratified by the miners as the network
upgrade activates.

**Furthering Decentralization**

ZF or FPF SHALL conduct periodic (e.g. semiannual or annual) reviews of the
organizational structure, performance, and effectiveness of the ZCG program and
committee, taking into consideration the input and recommendations of the ZCG
Committee. As part of these periodic reviews, ZF or FPF MUST commit to
exploring the possibility of transitioning ZCG into an independent organization
if it is economically viable and it aligns with the interests of the Zcash
ecosystem and prevailing community sentiment.

In any transition toward independence, priority SHALL be given to maintaining
or enhancing the decentralization of the Zcash ecosystem. The newly formed
independent organization MUST ensure that decision-making processes remain
community-driven, transparent, and responsive to the evolving needs of the
Zcash community and ecosystem. In order to promote geographic decentralization,
the new organization SHOULD establish its domicile outside of the United

> States.

Reviewer 4: Can you confirm that the intention here is for the ZIP draft to constrain that ZCG be transitioned to an org that is independent of _both_ ZF and FPF (rather than transitioned from being administered by ZF to being administered by FPF)?

Separately, the spec cannot have "e.g." as a SHALL requirement. The period needs to be decided (but that can be a parameter for the purposes of today's deadline).

Author: Correct, the intention is for ZCG to transition from being a committee under either ZF or FPF, to being an independent organization if it is economically viable and it aligns with the interests of the Zcash ecosystem and prevailing community sentiment. 

i think we can just remove the e.g completely from the sentence with a SHALL.

Author: I've removed the parenthetical that stated "e.g. semi-annual or annual".

Author: @str4d I've made the updates so that the ZIP only reflects FPF.
